docs/_static/css/mandoc.css

```diff
@@ -0,0 +1,251 @@
+/* $Id: mandoc.css,v 1.46 2019/06/02 16:57:13 schwarze Exp $ */
+/*
+ * Standard style sheet for mandoc(1) -Thtml and man.cgi(8).
+ *
+ * Written by Ingo Schwarze <schwarze@openbsd.org>.
+ * I place this file into the public domain.
+ * Permission to use, copy, modify, and distribute it for any purpose
+ * with or without fee is hereby granted, without any conditions.
+ */
+
+/*
+ * Edited by George Melikov <mail@gmelikov.ru>
+ * to be integrated with sphinx RTD theme.
+ */
+
+/* override */
+.man_container code {
+	overflow-x: initial;
+}
+
+/* OpenZFS styles */
+.head {
+	max-width: 640px;
+	width: 100%;
+}
+.head .head-vol {
+	text-align: center;
+}
+.head .head-rtitle {
+	text-align: right;
+}
+.foot td {
+	padding: 1em;
+}
+
+/* Sections and paragraphs. */
+
+.manual-text {
+		margin-left: 0em; }
```

Comment:
There is some goofy formatting (of the CSS itself) / whitespace starting here.

Reply:
It's an upstream formatting, i'd prefer to leave it as is to easily backporting in future, added same comment.